book/tutorials/day_3.md

```diff
@@ -0,0 +1,4 @@
+# Day 3
+
+```{tableofcontents}
+```
```

Comment:
Looks like this file snuck in when porting the notebook from last year. Should be safe to remove